framework/math/spatial_discretization/finite_volume/finite_volume.cc

```diff
@@ -8,15 +8,12 @@
 #include "framework/mpi/mpi.h"
 #include "framework/mpi/mpi_utils_map_all2all.h"
 
-#define sc_int64 static_cast<int64_t>
-
-#define MappingError                                                                               \
-  "SpatialDiscretization_FV::OrderNodes: "                                                         \
-  "Error mapping neighbor cells"
-
 namespace opensn
 {
 
+static std::string mapping_error =
+  "SpatialDiscretization_FV::OrderNodes: Error mapping neighbor cells";
+
```

Comment:
It looks like this is only used in `OrderNodes`. Should we define this inside the routine instead?